applications/search/src/redux/modules/informationModels.js

```diff
@@ -1,8 +1,13 @@
 import _ from 'lodash';
 import qs from 'qs';
```

Comment:
Probably not in use

Reply:
lint

Reply:
det er i bruk til å generere en kompakt nøkkel. JSON.stringify kunne også funke, menn det gir litt lengere nøkkel... skal vi fikse?